boot/zephyr/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  description: mcuboot test build sample
+  name: mcuboot
+
+tests:
+  sample.bootloader.mcuboot:
+    tags: bootloader_mcuboot
+    platform_whitelist: nrf52840_pca10056 frdm_k64f
```

Comment:
The whitelist doesn't contain pca10059. Is that ok? Will it get built for this board with CDC ACM recovery enabled?

Reply:
It will get built with CDC ACM recovery, yes, I tested it locally.

Reply:
BTW - build for pca10059 without CDC ACM would be almost identical as for pca10056 (only different flash layout).